plugins/java-decompiler/engine/README.md

```diff
@@ -14,8 +14,8 @@ Fernflower is licenced under the Apache Licence Version 2.0. It can be found her
 
 java -jar fernflower.jar [-<option>=<value>]* [<source>]+ <destination>
 
-* means 0 or more times
-+ means 1 or more times
+\* means 0 or more times\
```

Comment:
Now that I look at the rich diff, I see it's not necessary.
I thought that newlines were ignored at the end of lines, except when there's two of them (to create a new paragraph).
The \ was to force a \<br\> element but it seems that's inserted anyway.